tools/accuracy_checker/accuracy_checker/adapters/image_processing.py

```diff
@@ -213,3 +213,26 @@ def process(self, raw, identifiers=None, frame_meta=None):
             results.append(SuperResolutionPrediction(identifier, sr_img))
 
         return results
+
+
+class SuperResolutionGray(ImageProcessingAdapter):
+    __provider__ = 'super_resolution_gray'
+    prediction_types = (SuperResolutionPrediction, )
```

Comment:
do we actually need specific  adapter or usual super_resolution can work here?

Reply:
No, model works correct with super_resolution adapter. New adapter has been removed